src/modeler/AutoRest.Swagger/Validation/Extensions/NextLinkPropertyMustExist.cs

```diff
@@ -22,6 +23,18 @@ public class NextLinkPropertyMustExist : PageableExtensionRule
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
         /// <summary>
+        /// What kind of open api document type this rule should be applied to
+        /// </summary>
+        public override ServiceDefinitionDocumentType ServiceDefinitionDocumentType => ServiceDefinitionDocumentType.ARM;
```

Comment:
if the extension is used, then this should apply, so I think it should be default